### PR TITLE
Added Images Component

### PR DESCRIPTION
**Pull request by Shah-Arsalan**
I added the images component

### CODE REVIEW

**Review by lomesshh — APPROVED** [review]

**Review by Hnh013 — APPROVED** [review]
Looks good images are responsive ! Please align them in a cleaner way from later

File: ImageFolder/images.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/ImageFolder/images.css">
```
Comment (Rohanmond) [inline ImageFolder/images.html:7]: image.css is not inside ImageFolder...it is inside of components folder. Change the link. Except this everything looks good!

**Review by Rohanmond — APPROVED** [review]

**Review by vinc3nati — APPROVED** [review]
Good to proceed further!

File: ImageFolder/images.html (hunk 1 above)
Comment (Shah-Arsalan) [reply ImageFolder/images.html:7]: Actually that folder is imported . It's not part of the component library . I will delete it . Thanks!

**Pull request merged**